### PR TITLE
Improve Template Manager UI

### PR DESCRIPTION
**Pull request by marmarta**
Changed Template Manager UI to be more friendly.
Added a lot of explanations. Functionally,
added checking for obsolete templates.

### CODE REVIEW

**Comment by marmarta** [comment]
Should I base this on release4.2 or on main, @marmarek ?

**Comment by marmarek** [comment]
main

File: rpm_spec/qmgr.spec.in [hunk 1]
```diff
@@ -100,6 +100,7 @@ rm -rf $RPM_BUILD_ROOT
 
 %{python3_sitelib}/qubesmanager/resources_rc.py
 
+%{python3_sitelib}/qubesmanager/ui_templatemanger2.py
```
[the remark this answers is not in the document]
Comment (marmarta) [reply rpm_spec/qmgr.spec.in:103]: yes, good catch, no point keeping it

**Comment by marmarta** [comment]
>     5. Clicking "Repository settings" should go to "updates" tab, and ideally scroll to the templates section. Or at least say somewhere that user should go there.
> 
There is no good place for the info, so I opened https://github.com/QubesOS/qubes-issues/issues/9530 and will work on that right after this one.

**Comment by marmarek** [comment]
Included in #387

**Pull request closed without merging**